backend/e4h-services/asset-registry/README.md

```
@@ -0,0 +1,18 @@
+
+## Overview  
+This server was generated by the [swagger-codegen](https://github.com/swagger-api/swagger-codegen) project.  
+By using the [OpenAPI-Spec](https://github.com/swagger-api/swagger-core), you can easily generate a server stub.  
+This is an example of building a swagger-enabled server in Java using the SpringBoot framework.  
+
+The underlying library integrating swagger to SpringBoot is [springfox](https://github.com/springfox/springfox)  
+
+Start your server as an simple java application  
+
+You can view the api documentation in swagger-ui by pointing to  
+http://localhost:8080/  
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)